test/switchlevels.c

```diff
@@ -121,6 +122,12 @@ int main(int argc, char **argv) {
     int window_bits = MAX_WBITS + 16;
     PREFIX3(stream) strm;
 
+
+    if (argc <= 1) {
```

Comment:
argc cannot be 0, it's always at least 1, as the process name is the first argument.

Reply:
Fixed.